dds_cli/options.py

```diff
@@ -4,6 +4,9 @@
 import click
 from dds_cli.utils import multiple_help_text
 
+
+# Options
+
```

Comment:
I don't see any reason to have this comment. We have the module doc string on top as well as the file name to tell us that this file contains options.
```suggestion
```